kubernetes/charts/weblogic-domain/templates/_traefik-security.tpl

```diff
@@ -6,7 +6,7 @@
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
-  name: {{ .Release.Name }}-{{ .clusterName | lower }}-traefik
+  name: {{ .Release.Name }}-{{ .clusterName | lower | replace "_" "-" }}-traefik
```

Comment:
It would be better to write a helm function that converts a string to a valid k8s resource name, but I don't think it's worth holding up the review for it.